src/jdk.jartool/share/classes/sun/tools/jar/GNUStyleOptions.java

```diff
@@ -254,7 +264,8 @@ enum OptionType {
         CREATE("create"),
         CREATE_UPDATE("create.update"),
         CREATE_UPDATE_INDEX("create.update.index"),
-        OTHER("other");
+        OTHER("other"),
+        EXTRACT("extract");
```

Comment:
Is the order number important in this non-public `enum`? If not, I would expect `OTHER` to remain the last/highest entry.

Reply:
Hello Christian, I've now updated the PR to keep `OTHER` as the last enum value.